rllib/algorithms/algorithm.py

```diff
@@ -2233,6 +2233,23 @@ def resource_help(cls, config: Union[AlgorithmConfig, AlgorithmConfigDict]) -> s
             f"The config of this Algorithm is: {config}"
         )
 
+    @override(Trainable)
+    def get_auto_filled_metrics(
+        self,
+        now: Optional[datetime] = None,
+        time_this_iter: Optional[float] = None,
+        debug_metrics_only: bool = False,
+    ) -> dict:
+        autofilled = super().get_auto_filled_metrics(
+            now, time_this_iter, debug_metrics_only
+        )
+        # If `config` key is no dict (but AlgorithmConfig object) ->
+        # make sure, it's a dict to not break Tune APIs.
+        if not isinstance(autofilled["config"], dict):
```

Comment:
nit: can be more defensive here to see if config is present.

Reply:
added